ajax/libs/openui5-sap.ui.core/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@openui5",
+  "filename": "sap-ui-core.js",
+  "description": "OpenUI5 Core & Runtime - OpenUI5 Core Library sap.ui.core",
+  "license": "Apache-2.0",
+  "keywords": [
+    "openui5",
+    "sapui5",
+    "ui5"
+  ],
+  "author": "SAP SE <*@sap.com>",
+  "homepage": "https://openui5.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/openui5.git"
+  },
+  "npmName": "@openui5/sap.ui.core",
+  "npmFileMap": [
+    {
+      "basePath": "sap.ui.core/src",
+      "files": [
+        "sap-ui-core.js"
```

Comment:
As mentioned in https://github.com/cdnjs/cdnjs/pull/11789#issuecomment-328535263 this is not sufficient.
Also, the npm packages currently don't contain bundles at all.

Reply:
Hey @matz3 - where can we pull the bundles in from, do you have tagged release on GitHub that contain the bundled versions?

Adittionally, could you provide a full list of what files we should be including here?

Reply:
No, we don't have the bundles within npm or Github right now. We consider adding them to the npm packages or to provide additional packages in future.

The only way would be to build them manually or fetch them from our CDN (https://openui5.hana.ondemand.com).

To be on the safe side you would need to include all files within the "src" folder.
In addition to that, just adding the sap.ui.core library won't be sufficient, as more libraries are required to use OpenUI5 (see https://www.npmjs.com/org/openui5).

Also, the URL paths will be different than expected by default so users need to configure paths to the individual libraries in order to properly load them.

While I really like the idea to have a central CDN to host all kind of JavaScript frameworks and libraries I don't quite see the benefit of adding OpenUI5 as there is already a CDN and the steps to add it are not as straightforward as for other, rather simple libraries just consisting of one JS file.

Reply:
Makes sense. I think I agree that adding it here is more work than the benefit it gives 👍